lib/jekyll-browsersync/command.rb

```diff
@@ -42,20 +42,13 @@ def self.process(args = [], options = {})
           args << "--no-open" unless config["open"]
           cmd = "#{cli} start #{args.join(" ")}"
 
-          if `#{cli} --version 2>/dev/null`.empty?
```

Comment:
some systems will report an error if the command can't be found, so this will pass. Maybe this should be a private method to check existence based on OS type?